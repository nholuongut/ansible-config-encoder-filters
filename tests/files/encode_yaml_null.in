var1:
var2: null
