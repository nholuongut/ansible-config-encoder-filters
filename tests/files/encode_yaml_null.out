var1: null
var2: null
